shub/utils.py

```diff
@@ -830,6 +830,6 @@ def create_scrapinghub_yml_wizard(conf, target='default', image=None):
     if image or (image is None and _detect_custom_image_project()):
         repository = click.prompt(
             "Image repository (leave empty to use Scrapinghub's repository)",
-            default=True, show_default=False)
+            default=repository, show_default=False)
```

Comment:
Could you elaborate why we are changing `True` to `None` here, how this affects behavior? Is this something we need to do before rebranding? (I’m just wary of behavior changes breaking code in corner cases)

Reply:
This was an incorrect change. `shub` actually depends on `default=True` as a default.